applications/luci-app-packet-capture/root/usr/sbin/packet_capture_stop

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+pgrep tcpdump | xargs kill -9
```

Comment:
... and use that pid file here to avoid killing unrelated processes.

```bash
pid=$(cat /var/run/packet_capture.pid)
if [ -n "$pid" ] && grep -sq tcpdump "/proc/$pid/cmdline"; then
    kill -TERM $pid
fi
rm -f /var/run/packet_capture.pid
```

Reply:
Actually tcpdump process has to be terminated to that is does not become zombie